composer.json

```diff
@@ -127,6 +186,9 @@
         "patches": {
             "drupal/core": {
                 "Add contextual link to blocks in layout builder": "https://www.drupal.org/files/issues/2019-09-06/contextual_links_with_LB-3020876-18_0.patch"
+            },
+            "drupal/console": {
+                "Fix spelling error in plugin.php.twig": "https://github.com/ooneill547/drupal-console/commit/ef82d2548c34d21f5288ee59b322bc10dbdf753c.patch"
```

Comment:
If you get a few spare minutes, I'd consider opening a pull request at https://github.com/hechoendrupal/drupal-console to get some credit

Reply:
Already done. I don't think it's been accepted yet.